tests/data_reader.py

```diff
@@ -174,11 +221,11 @@ def read_errors_section(self):
         # The 'ERRORS' section is optional.
         tag_line = self.read_line()
         if tag_line is None:
-            return []
-        (tag, _) = self.parse_tag_line(tag_line)
+            return None, None
+        (tag, error_flags) = self.parse_tag_line(tag_line)
```

Comment:
A bit confused why `error_flags` is extracted, instead of just being ignored as was originally. It does not seem like there were any flags added to the `ERRORS` section, so this flag is never used. The consequence is slightly more complicated return values and more complicated handling in the calling code.

Reply:
reverted in be3a37a